README.md

```diff
@@ -43,6 +43,13 @@ Start the webpack development server:
 npm start
 ```
 
+Start the remote redux dev tools server:
+
+```
+npm run remotedev
+```
+Use the [redux devtools extension](https://github.com/zalmoxisus/redux-devtools-extension) to inspect and dispatch actions.
```

Comment:
I think it's worth adding first that you can just go to `http://localhost:9090/` in the browser (a bit easier than installing a Chrome extension).